packages/codebytes/src/libs/eventTracking.ts

```diff
@@ -3,6 +3,7 @@ import { createTracker } from '@codecademy/tracking';
 const IS_DEV = process.env.NODE_ENV === 'development';
 
 // TODO: confirm tracking details and implementation DISC-447
+// the window.location.origin will match what the iframe url is which in forums is https://www.codecademy.com
```

Comment:
i thought it would be
```suggestion
// the window.location.origin will match what the iframe url is which in forums is https://www.codecademy.com/codebyte-editor
```
?

Reply:
Technically that is the url but the origin of that url is just `https://www.codecademy.com/` 

Maybe to be super explicit, I can put, "The iframe url in the forums is https://www.codecademy.com/codebyte-editor. The window.location.origin will be derived from that url and will be https://www.codecademy.com."

Reply:
then why do we even need the ternary?

Reply:
Because we want it to work on non-production urls. The ternary/fallback is just to prevent it from erroring when this code is run on the server which it will be when it gets statically rendered in Next.js

Reply:
ohh then i dont think this comment is helpful haha, we can probably just close the pr. i was just confused what this ternary was for!